### PR TITLE
feat: submit master data form

### PR DESCRIPTION
**Pull request by Ibwedagama**
### Related
- [S55A2 - Add Master Data Layanan Form 1](https://app.clickup.com/t/860pxrrhy)
- [S55A7 - Add Master Data Layanan Form 2](https://app.clickup.com/t/860q03hen)
- [S55A8 - Add Master Data Layanan Form 3](https://app.clickup.com/t/860q03hm9)

### Changes
- create `SubmitConfirmation` modal component
- add submit form functionality

### Preview

https://user-images.githubusercontent.com/33661143/224662026-53aa6ad0-a00f-4455-bb0b-78e3718ca6b9.mov

### Evidence
title: feat submit master data form
project: Portal Jabar
participants: @Ibwedagama @agunghide @yoslie @doohanas

### CODE REVIEW

**Review by agunghide — APPROVED** [review]

File: src/components/Services/MasterData/Form/SubmitConfirmation.vue [hunk 1]
```diff
@@ -0,0 +1,569 @@
...
+
+        <div class="px-4 py-3 bg-white even:bg-[#FCFCFC]">
+          <p class="font-roboto text-base leading-6 text-[#627798]">
+            Status Operasional:
+          </p>
+          <p
+            class="w-fit font-lato font-bold text-[14px] leading-[23px] capitalize mt-2"
+            :class="{
+              'bg-green-50 text-green-700 px-[10px] rounded-[5px] font-bold ' : operationalStatus === 'ACTIVE',
+              'bg-gray-200 text-gray-600 px-[10px] rounded-[5px] font-bold ' : operationalStatus === 'NOT-ACTIVE'
```
Comment (yoslie) [inline src/components/Services/MasterData/Form/SubmitConfirmation.vue:74]: very minor: can move static class to line 71 (`px-[10px] rounded-[5px] font-bold`) just to make it simple in dynamic class

File: src/components/Services/MasterData/Form/SubmitConfirmation.vue [hunk 2]
```diff
@@ -0,0 +1,569 @@
...
+
+        <div class="px-4 py-3 bg-white even:bg-[#FCFCFC]">
+          <p class="font-roboto text-base leading-6 text-[#627798]">
+            Teknis Layanan:
+          </p>
+          <p
+            :class="{
+              'font-lato font-bold text-[14px] leading-[23px] capitalize mt-2' : true,
+              'w-fit rounded-[6px] bg-[#E3F0FF] text-blue-800 px-2 py-[2px]': technical === 'ONLINE',
+              'w-fit rounded-[6px] bg-gray-200 text-gray-600 px-2 py-[2px]': technical === 'OFFLINE'
```
Comment (yoslie) [inline src/components/Services/MasterData/Form/SubmitConfirmation.vue:89]: this one too ✌️

**Review by yoslie — APPROVED** [review]

**Review by doohanas — APPROVED** [review]

File: src/components/Services/MasterData/Form/SubmitConfirmation.vue (hunk 1 above)
Comment (Ibwedagama) [reply src/components/Services/MasterData/Form/SubmitConfirmation.vue:74]: okay @yoslie thanks for pointing that out 👍🏼

Comment (Ibwedagama) [reply src/components/Services/MasterData/Form/SubmitConfirmation.vue]: [fixed here](https://github.com/jabardigitalservice/portal-jabar-cms/pull/282/commits/f38c03de29e002ae9fdebc7456238a7a91cb7c86) @yoslie thank you

**Pull request merged**